configs/emcts1/victimplay.cfg

```diff
@@ -0,0 +1,14 @@
+@include ../../engines/KataGo-raw/cpp/configs/training/selfplay1.cfg
+@include base.cfg
+
+# These seem to be necessary to prevent KataGo from crashing, ONLY because we often
+# run victimplay with quite low visit counts for the victim relative to what's usually
+# used in standard selfplay.
+cheapSearchProb = 0                 # Should we set this to the default value of 0.75?
+handicapAsymmetricPlayoutProb = 0
+normalAsymmetricPlayoutProb = 0
+reduceVisits = false                # Should we set this to the default value of true?
+
+# EMCTS doesn't currently support this
+switchNetsMidGame = false
+useAuxPolicyTarget = false
```

Comment:
Or actually EMCTS does support useAuxPolicyTarget. We saw in some runs that this helps with learning, though we haven't tested this against hardened adversaries.

Reply:
Should we toggle it to true by default then?

Reply:
Sorry, I know it supports `useAuxPolicyTarget`, the comment was only meant to apply to `switchNetsMidGame`

Reply:
But you do need to specify _some_ value for `useAuxPolicyTarget` explicitly and I just set it to `false` since it seems like that's what we've been using